ocs_ci/ocs/node.py

```diff
@@ -444,6 +457,15 @@ def get_node_resource_utilization_from_oc_describe(nodename=None, node_type='wor
             'memory': int(mem[0])
         }
 
+    if print_table:
+        usage_memory_table = PrettyTable()
+        usage_memory_table.field_names = [
+            "Node Name", "CPU USAGE oc_describe", "Memory USAGE oc_describe"
+        ]
+        for node, util_node in utilization_dict.items():
+            usage_memory_table.add_row([node, f'{util_node["cpu"]}%', f'{util_node["memory"]}%'])
+        log.info(f'\n{usage_memory_table}\n')
+
```

Comment:
the code here is a dup of https://github.com/red-hat-storage/ocs-ci/pull/2262/files#diff-ffa18f30e526ae7b2498579992b406efR412-R419.

Could you please create a function for that?